qiskit/backends/ibmq/ibmqprovider.py

```diff
@@ -128,3 +161,9 @@ def _discover_remote_backends(self):
             ret[config['name']] = IBMQBackend(configuration=config, api=self._api)
 
         return ret
+
+
+def least_busy(backends):
```

Comment:
Can we move this to the wrapper, and update the examples so they make use of this function? It seems to be a convenience function (and one that will be much appreciated for the tutorials :smile:  ), but not a function that is used by this provider or the rest of the SDK itself.

Reply:
The reason I put it here is because it is specific to the ibmqprovider. Another provider (e.g. local) may not have a queue system, hence `least_busy` will not make sense.

Reply:
Can you check how does it play with the idea of the levels? I honestly don't know if it is problematic that they include the extra `import qiskit.backends.ibmq.ibmqprovider ....` on top (if it is, might be just solved by importing them from the wrapper).